sub_scripts/testing_process.sh

```diff
@@ -342,9 +342,9 @@ CHECK_URL () {
 					fi
 
 					# Print the first 20 lines of the page
-					ECHO_FORMAT "Extract of the page:\n" "white"
+					ECHO_FORMAT "Extract of the page:\n" "white" >> "$test_result"
 					echo -en "\e[37m"	# Write in 'light grey'
-					lynx -dump -force_html "$script_dir/url_output" | head --lines 20 | tee --append "$test_result"
+					lynx -dump -force_html "$script_dir/url_output" | head --lines 20 >> "$test_result"
```

Comment:
I'd like to keep it, because in some case there are false negative errors cf. [Installation in a sub path](https://ci-apps-unstable.yunohost.org/ci/job/711)

```
Real url: https://sous.domain.tld/path/
HTTP code: 620
Title of the page: Dotclear - Error
Extract of the page:
Dotclear

Database problem

   Something went wrong while trying to read the database.


Test url: sous.domain.tld/path/
Real url: https://sous.domain.tld/path/
HTTP code: 620
Title of the page: Dotclear - Error
Extract of the page:
Dotclear

Database problem

   Something went wrong while trying to read the database.
```

In a perfect world we want to detect this and remove false negative errors, but we can't handle all specific errors

Reply:
Yes indeed

Later in the code (at least in that PR but not the other I believe :s ) I added : 

```
    if [[ curl_error -ne 0 ]]
    then
        warning "$test_url_details"
        warning "Page title: $page_title"
        warning "Page extract: $page_extract"
    fi
```

with the idea of displaying the info only if it's not what's expected ... though now that you mention it, the current code only assumes it's a curl_error if it's 4xx or 5xx .. maybe it should also consider 6xx codes (c.f your example) (though I don't even understand how that code exists in the first place but anyway)